rosetta-source/src/main/rosetta/product-collateral-type.rosetta

```diff
@@ -111,6 +111,7 @@ type CollateralCriteriaBase: <"Represents a set of criteria used to specify and
 type EligibleCollateralCriteria extends CollateralCriteriaBase: <"Represents a set of criteria used to specify eligible collateral.">
 
 	treatment CollateralTreatment (1..1) <"Identifies the treatment of specified collateral, e.g., haircuts,holding limits or exclusions.">
+    partyCriteriaAppliesTo Party (1..*) <"Identifies the parties to which the specified eligible collateral and treatment applies to. e.g. Party A or Party B (Pledgor or Receiver)">
```

Comment:
Is this referring to a Party by name or by Role? From your description it looks like by role in which case would this be type partyRole instead? Happy adding the party at this level but I don't think it should be mandatory i.e. cardinality 0..* rather than 1..*, as there may be no need to identify a party.

Reply:
replied as a comment